### DISCUSSION TITLE
flatseal: How can it be firejailed if it's a flatpak?

### DISCUSSION THREAD

**Discussion by remyabel2** [netblue30/firejail#4938] [Q&A] [answered]
As I understand it Firejail is not meant to be combined with Flatpak, yet I see some profiles like https://github.com/netblue30/firejail/blob/9b1daa6b3abecfa0b270e9202106575cd49aeacc/etc/profile-a-l/com.github.tchx84.Flatseal.profile which is a Flatpak only app. What is the intended usage/usecase of these profiles?

**Comment by rusty-snake** [comment, marked as answer]
> which is a Flatpak only app

Source?

https://aur.archlinux.org/packages/flatseal
~https://github.com/tchx84/Flatseal#build-it-yourself~

> What is the intended usage/usecase of these profiles?

To run flatseal in a firejail sandbox :thinking:
<sup>if you installed it via AUR, *raw* meson, ...</sup>

> Firejail is not meant to be combined with Flatpak

It can not.

**Reply by remyabel2** [reply]
Not sure what the purpose of the terseness is for but alright.

> Source?

I was not aware of the AUR package and you linked to the Flatpak builder instructions which is exactly what I read before coming here.

> To run flatseal in a firejail sandbox

My question was obviously implied to mean "given the (incorrect) assumption that it is a Flatpak only app, how would one run it with Firejail?"

> It can not.

I know. I read multiple posts stating that before coming here. Hence my question.

Anyway my question has been answered.
